packages/terra-toolkit-docs/src/terra-dev-site/tool/jest-config-terra/About.a.tool.mdx

```diff
@@ -0,0 +1,95 @@
+import { Badge } from '@cerner/jest-config-terra/package.json?dev-site-package';
+
+<Badge />
+
+# Jest Config Terra
+
+This package provides Terra's recommended jest configuration.
+
+Features:
+* An extended Jsdom test environment that:
+  * provides a mock implementation for `matchMedia`.
+  * provides a mock implementation for `scrollIntoView`.
+  * Sets the dir tag on the html element.
+* Clears mocks after every test by default.
+* Sets up code coverage.
+* Specifies test matchers to not match on terra-dev-site files.
+* Supports mono repos using a 'packages' directory.
+* Maps css, scss, and svg files to the [identity-obj-proxy](https://github.com/keyz/identity-obj-proxy).
+* Maps terra-aggregate-translations locale files to a mock implementation. This removes the need to run terra-aggregate-translations prior to jest testing.
+
+
+## What is Jest
+
+[Jest](https://eslint.org/) is a "Jest is a delightful JavaScript Testing Framework with a focus on simplicity".
```

Comment:
```suggestion
"Jest is a delightful JavaScript Testing Framework with a focus on simplicity" - [Jest](https://eslint.org/)
```

Reply:
Should Jest link to... jest?

https://jestjs.io/

Reply:
lol, yeah that makes sense. You can guess where I copied this about page from :)